src/Common/MiniTerminal.cpp

```diff
@@ -133,12 +124,12 @@ void MiniTerminal::process()
             m_input[m_writeIndex] = '\0';
         }
         /* Remove the last character from command line? */
-        else if ((ASCII_DEL == buffer[idx]) ||
-                 (ASCII_BS == buffer[idx]))
+        else if ((ASCII_DEL == currentChar) ||
+                 (ASCII_BS == currentChar))
         {
             if (0 < m_writeIndex)
             {
-                char removeSeq[] =
+                static char removeSeq[] =
```

Comment:
That indeed saved 20 bytes of code space.. Looks like dynamicly setting it up is way more expensive then storing int in .data segment.

Reply:
I guess the setup on the stack take some bytes. Could we add additional "const"?

Reply:
ok